src/LodeRunner.API/src/Controllers/TestRunsController.cs

```diff
@@ -132,6 +133,10 @@ public async Task<ActionResult> CreateTestRunConfig([FromBody, SwaggerRequestBod
                 return await ResultHandler.CreateCancellationInProgressResult();
             }
 
+            // Explicitly set CompletedTime to null, in case CompletedTime is set in TestRunPayload
+            // TODO: Would have to create a new object to make AutoMapper ignore CompletedTime for Post only. Can we do better? maybe even using [Bind]

```

Comment:
@gortegaMS @joheec
This slightly feels like a "small" hack.
Hence the TODO: Comment.